### DISCUSSION TITLE
Verification issues

### DISCUSSION THREAD

**Discussion by TesDevelopment** [community/community#53638] [GitHub Education]
I've uploaded images of my portal and transcripts and it keeps telling me that my profile is not complete. My profile is complete and even though I am not comfortable with my full legal name being public I tried that aswell, it gave me the same response.

**Comment by samus-aran** [comment]
Hi @TesDevelopment 👋 

Thank you for feedback and raising your concerns! We will pass this along to our internal team.

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.
We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying. 👈 

While we do take our Community’s feedback seriously we are also mindful to ensure that bad actors impersonating students don't jeopardize our ability to provide academic benefits of value. One aspect of this protection is momentarily compare the evidence provided against your GitHub account information.

I can confirm that once you have submitted your application you can revert this information back. For more information on how GitHub processes and uses personal data, please see the [GitHub Privacy Statement](https://docs.github.com/en/site-policy/privacy-policies/github-privacy-statement).
